### ISSUE TITLE
Process Object enhancements

### ISSUE THREAD

**Issue by athiasjerome** [CybOXProject/schemas#375]
The Process_Object should be enhanced, offering the possibility to define more direct relationships with other CybOX objects.
For example:
ProcessObj:ImageInfoType should offer the possibility of refering to a File Object, in addition to just provide a File_Name

**Comment by ikiril01** [comment]
I agree. We've known about the Image_Info issue for a while, as in #38. Are there other enhancements in the way of more direct relationships that could be made?
